### PR TITLE
#46 Add constructor tests for TopicDeleteResult

### PR DESCRIPTION
**Pull request by rdodiya**
### Summary
Constructor tests for the TopicDeleteResult was added in the ProtocolLayerDataCreationTests class
A new test method i.e testTopicDeleteResult was added in the ProtocolLayerDataCreationTests class that implements all the needed tests for the TopicDeleteResult.

### CODE REVIEW

File: hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java [hunk 1]
```diff
@@ -727,4 +728,17 @@ void testTopicSubmitMessageResultCreation() {
         Assertions.assertThrows(NullPointerException.class, () -> new TopicSubmitMessageResult(null, validStatus));
         Assertions.assertThrows(NullPointerException.class, () -> new TopicSubmitMessageResult(validTransactionId, null));
     }
+    
+    @Test
+    void testTopicDeleteResult() {
```
Comment (Ndacyayisenga-droid) [inline hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java:733]: @rdodiya looks good but change `testTopicDeleteResult()` method name to `testTopicDeleteResultCreation()` for consistence in the naming convetion.

File: hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java [hunk 2]
```diff
@@ -727,4 +728,17 @@ void testTopicSubmitMessageResultCreation() {
...
+
```
Comment (Ndacyayisenga-droid) [inline hedera-base/src/test/java/com/openelements/hedera/base/test/ProtocolLayerDataCreationTests.java:734]: Delete this empty line as well

**Comment by rdodiya** [comment]
Hi @Ndacyayisenga-droid , I have updated the changes, please review it.

**Comment by Ndacyayisenga-droid** [comment]
> Hi @Ndacyayisenga-droid , I have updated the changes, please review it.

LGTM

**Review by Ndacyayisenga-droid — APPROVED** [review]

**Pull request merged**